.github/workflows/operator-integration-test.yml

```diff
@@ -104,31 +90,51 @@ jobs:
           kubectl delete pods --all -n default
           sleep 5
           kubectl wait --for=condition=Ready pod --all -n default
-          
           cd integration-tests/manifests/cmd
           go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json
           kubectl delete instrumentation sample-instrumentation
 
-      - name: Test for default instrumentation resources for python and java
+      - name: Test for default instrumentation resources for dotnet
+        run: |
+          cat integration-tests/dotnet/sample-deployment-dotnet.yaml
+          kubectl apply -f integration-tests/dotnet/sample-deployment-dotnet.yaml
+          sleep 5
+          kubectl wait --for=condition=Available deployment/nginx -n default
+          kubectl get pods -A 
+          kubectl describe pods -n default
+          go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/dotnet/default_instrumentation_dotnet_env_variables.json
+
+      - name: Test for defined instrumentation resources for dotnet
         run: |
-          kubectl apply -f integration-tests/python-java/sample-deployment-python-java.yaml
+          kubectl apply -f integration-tests/manifests/sample-instrumentation.yaml
           kubectl delete pods --all -n default
           sleep 5
-          kubectl wait --for=condition=Ready pod --all -n default
-          
-          go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/python-java/default_instrumentation_python-java_env_variables.json
+          kubectl wait --for=condition=Ready pod --all -n default  
+          cd integration-tests/manifests/cmd
+          go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json
+          kubectl delete instrumentation sample-instrumentation
+
+
+      - name: Test for default instrumentation resources for python java dotnet
```

Comment:
imo this should just be `Test for default instrumentation resources for all languages` and then you don't have to pile on each language. Same for the directory name `integration-tests/python-java-dot-net` -> `integration-tests/all-languages`

Reply:
Ahh yes this will be useful when we add more lanugages, made the change.